src/EventStore.Core.Tests/Services/Transport/Http/build_requested_url_should.cs

```diff
@@ -52,43 +52,43 @@ public void with_advertise_ip_and_http_port_set_both_host_and_port_is_changed()
         }
 
         [Test]
-        public void with_port_forward_header_only_port_is_changed()
+        public void with_port_forward_header_only_nothing_is_changed()
```

Comment:
Any particular reason you changed the description of these tests to "nothing_is_changed"? They did change though?